src/lib.js

```diff
@@ -68,42 +94,37 @@ async function getOmgevingInfo(data) {
   return data;
 }
 
-
-const callchain = [
-  wrapApiCall(getFullObjectData),
-  wrapApiCall(getOmgevingInfo)
-]
-
-//user-provided function for featureQuery to format request URL
-//featureQuery will call this function with the following arguments:
-// baseUrl string: the base url of the API (from config)
-// xy: the x,y of the clicked point on the map (longitude, latitude).
 function requestFormatter(baseUrl, xy) {
   let xyRD = transformCoords.forward(xy);
   return `${baseUrl}${xyRD.x},${xyRD.y},50`
 }
 
-//user-provided function for featureQuery to parse Ajax response
-//we need:
-//* query (lat/lon)
-//* object
-//* omgeving:
-//  * dichtsbijzijnd adres
-//  * buurtnaam
-//  * buurtcode
-//  * stadsdeelnaam
-//  * stadsdeelcode
-//  * buurtcombinatienaam
-//  * buurtcombinatiecode
-//
-
-//So we need several of these request/responseformatter things.
-//And then we chain them together.
 function responseFormatter(res) {
-  let filtered = res.results.filter(x => x.hoofdadres === true);
+  let filtered;
+  try {
+    filtered = res.results.filter(x => x.hoofdadres === true);
+  } catch (e) {
```

Comment:
Ik zou eventueel checken of de `results` parameter bestaan en een error throwen als dat niet het geval is. Ik zou echter geen try catch gebruiken.

Reply:
Goed punt, aangepast